pkg/spanconfig/spanconfigjob/job.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package spanconfigjob
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/jobs"
+	"github.com/cockroachdb/cockroach/pkg/jobs/jobspb"
+	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
+	"github.com/cockroachdb/cockroach/pkg/sql"
+	"github.com/cockroachdb/errors"
+)
+
+type resumer struct {
+	job *jobs.Job
+}
+
+var _ jobs.Resumer = (*resumer)(nil)
+
+// Resume implements the jobs.Resumer interface.
+func (r *resumer) Resume(ctx context.Context, execCtxI interface{}) error {
+	execCtx := execCtxI.(sql.JobExecContext)
+	rc := execCtx.SpanConfigReconciliationJobDeps()
+	// TODO(zcfg-pod): Upcoming PRs will actually make use of these reconciliation
+	// dependencies.
+	_ = rc
+
```

Comment:
Is the idea that the implementation of reconciliation will happen here or is is the idea that there will be another injected piece of machinery that gets invoked?